libraries/InternalFileSytem/examples/Internal_StressTest/Internal_StressTest.ino

```diff
@@ -0,0 +1,130 @@
+/*********************************************************************
+ This is an example for our nRF52 based Bluefruit LE modules
+
+ Pick one up today in the adafruit shop!
+
+ Adafruit invests time and resources providing this open source code,
+ please support Adafruit and open-source hardware by purchasing
+ products from Adafruit!
+
+ MIT license, check LICENSE for more information
+ All text above, and the splash screen below must be included in
+ any redistribution
+*********************************************************************/
+
+#include <Adafruit_LittleFS.h>
+#include <InternalFileSystem.h>
+
+using namespace Adafruit_LittleFS_Namespace;
+
+/* This example perform a stress test on Little FileSystem on internal flash.
+ * There are 4 different thread sharing same loop() code with different priority
+ *    loop (low), normal, high, highest
+ * Each will open and write a file of its name (+ .txt). Task takes turn writing until timeout
+ * to print out the summary
+ */
+
+// timeout in seconds
+#define TIME_OUT      600
+
+uint32_t writeCount = 0;
+
+// the setup function runs once when you press reset or power the board
+void setup() 
+{
+  Serial.begin(115200);
+  while ( !Serial ) yield();   // for nrf52840 with native usb
+
+  Serial.println("Internal Stress Test Example");
+  yield();
+
+  // Initialize Internal File System
+  InternalFS.begin();
+
+  // Format
+  Serial.print("Formatting ... "); Serial.flush();
+  InternalFS.format();
+  Serial.println("Done"); Serial.flush();
+
+  // Create thread with different priority
+  // Although all the thread share loop() code, they are separated threads
+  // and running with different priorities
+
+  // Note: default loop() is running at LOW
+  Scheduler.startLoop(loop, 1024, TASK_PRIO_NORMAL, "normal");
+  Scheduler.startLoop(loop, 1024, TASK_PRIO_NORMAL, "normal");
+  Scheduler.startLoop(loop, 1024, TASK_PRIO_NORMAL, "normal");
```

Comment:
I use threadname as filename for each thread to write, we should name them differently even though they share the priority. Maybe n1, n2, n3. I will made another PR   to update the sketch